packages/perseus/src/widgets/__stories__/expression.stories.jsx

```diff
@@ -1,26 +1,61 @@
 // @flow
 import * as React from "react";
 
-import {RendererWithDebugUI} from "../../../../../testing/renderer-with-debug-ui.jsx";
+import {ItemRendererWithDebugUI} from "../../../../../testing/item-renderer-with-debug-ui.jsx";
+import KeypadContext from "../../keypad-context.js";
 import {
     expressionItem2,
     expressionItem3,
 } from "../__testdata__/expression_testdata.js";
 
+import TestKeypadContext from "./test-keypad-context-wrapper.jsx";
+
+import type {PerseusItem} from "../../perseus-types.js";
+
 type StoryArgs = {||};
 
 type Story = {|
     title: string,
 |};
 
+type WrappedKeypadContextProps = {|
+    item: PerseusItem,
+|};
+
+const WrappedKeypadContext = (props: WrappedKeypadContextProps) => {
+    return (
+        <TestKeypadContext>
+            <KeypadContext.Consumer>
+                {({keypadElement, setRenderer, scrollableElement}) => {
+                    return (
+                        <ItemRendererWithDebugUI
```

Comment:
I switched to ItemRendererWithDebugUI because ItemRenderer has `keypadElement={this.keypadElement()}` when rendering Renderer whereas RendererWithDebugUI does not - which was causing errors when trying to open the keypad.

I imagine I could have added it to RendererWithDebugUI, but I'm not totally sure I understand why we have both ItemRendererWithDebugUI and RendererWithDebugUI. Figured it was best to let sleeping dogs lie.